### PR TITLE
Migrate all imports to Next Images

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5929

### CODE REVIEW

**Review by Betree — CHANGES_REQUESTED** [review]
The first one I checked has a responsiveness regression. We need to make sure that the images styles & responsiveness are preserved. It's ok to not replace everything, but all replacements should be carefully tested.

image

**Comment by SudharakaP** [comment]
> The first one I checked has a responsiveness regression. We need to make sure that the images styles & responsiveness are preserved. It's ok to not replace everything, but all replacements should be carefully tested.
> 
> image

@Betree : Sorry I've missed this on mobile. I've applied a fix and also tested the others. Seems fine. We need to wrap this in a sizing container for different screen sizes since we need specific sizes for this (and similar) images.

File: components/collective-navbar/NavBarCategoryDropdown.js [hunk 1]
```diff
@@ -142,7 +138,7 @@ export const NavBarCategory = ({ category, collective }) => {
   return (
     <Flex>
       <Flex alignItems="center" mr={2}>
-        <IconIllustration src={categoryDetails.img || SECTIONS_CATEGORY_ICON.CONTRIBUTE} />
+        <Image width={32} height={32} src={categoryDetails.img || SECTIONS_CATEGORY_ICON.CONTRIBUTE} />
```
Comment (Betree) [inline components/collective-navbar/NavBarCategoryDropdown.js:141]:
```suggestion
        <Image width={32} height={32} src={categoryDetails.img || SECTIONS_CATEGORY_ICON.CONTRIBUTE} alt="" />
```

File: components/contribution-flow/ContributionFlowSuccess.js [hunk 2]
```diff
@@ -133,7 +132,11 @@ class ContributionFlowSuccess extends React.Component {
 
     return (
       <Flex flexDirection="column" justifyContent="center" p={2}>
-        {callsToAction.length <= 2 && <SuccessIllustration alt="" />}
+        {callsToAction.length >= 2 && (
+          <SuccessIllustration>
+            <Image width={216} height={152} src="/static/images/success-illustration.jpg" />
```
Comment (Betree) [inline components/contribution-flow/ContributionFlowSuccess.js:137]:
```suggestion
            <Image alt="" width={216} height={152} src="/static/images/success-illustration.jpg" />
```

File: components/expenses/ExpenseTypeRadioSelect.js [hunk 3]
```diff
@@ -86,38 +76,42 @@ const ExpenseTypeOptionContainer = styled.label`
...
+  [expenseTypes.RECEIPT]: '/static/images/receipt-animation-static.jpg',
+  [expenseTypes.GRANT]: '/static/images/grant.gif',
 };
 
 const ExpenseTypeOption = ({ name, type, isChecked, onChange }) => {
   const { formatMessage } = useIntl();
-  const illustrationSrc = illustrations[type] || receiptIllustration;
-  const staticIllustrationSrc = staticIllustrations[type] || receiptIllustrationStatic;
+  const illustrationSrc = illustrations[type] || '/static/images/receipt-animation.gif';
+  const staticIllustrationSrc = staticIllustrations[type] || '/static/images/receipt-animation-static.jpg';
```
Comment (Betree) [inline components/expenses/ExpenseTypeRadioSelect.js:93]: :no_entry_sign: Here the animation should only happen on hover

Peek 2022-09-16 10-26

**Review by Betree — COMMENTED** [review]
Looking good, some stuff to fix below but we're almost there

File: components/expenses/ExpenseTypeRadioSelect.js (hunk 3 above)
Comment (SudharakaP) [reply components/expenses/ExpenseTypeRadioSelect.js:102]: @Betree : Ops, I believe we need to handle this a bit differently than I thought. I just made a fix for this. 👍🏽

**Review by Betree — COMMENTED** [review]
Because the image is not preloaded there is a very noticeable delay when hovering in live envs. Either we can preload easily, or we should rollback this part to the previous version.

Peek 2022-10-21 11-45

**Comment by Betree** [comment]
Now that we have https://github.com/opencollective/opencollective-frontend/pull/8996, we'll need another pass to make sure the newly introduced `<Image>` match the latest API.

**Comment by Betree** [comment]
There are too many conflicts now, and there were some regressions in this PR: https://github.com/opencollective/opencollective-frontend/pull/8184#pullrequestreview-1150672334. We'll start fresh.

**Comment by Betree** [comment]
Actually, I'm gonna try to rebase and remove all the parts where we noticed a regression

**Review by Betree — APPROVED** [review]

**Pull request merged**